docs/guides/development/pack-maker-tools.md

```diff
@@ -0,0 +1,108 @@
+# Pack maker tools
+
+While the Pack Studio and Pack CLI are used to develop a Pack, the primary way to monitor and interact with a Pack once it is installed in a document is via the Pack maker tools. It is a pane along the bottom of a Coda document that provides information about the Packs installed in the doc that you have edit access to.
+
+
+## How to access
+
+There are a few different ways to open the Pack maker tools.
+
+=== "Doc menu"
+    Click the three dots icon to the right of the doc's name and select **Toggle Pack maker tools**.
```

Comment:
The reason I like this entry point is that it's always a short number of clicks away. The Pack panel is constantly being superseded by the sync table panel and others. That said, I don't find myself using it as often, likely because I didn't know it was there for a long time :-)